ChangeLog

```diff
@@ -4,69 +4,105 @@
 
 ### 3.7.0rc1 <- NOTE: the release version number will be 3.7.0 ###
 
-- introduced new server lists: Any Genre 3, Choral/Barbershop (#875),
-  also renamed Default to Any Genre 1, Any Genre to Any Genre 2,
-  and Classical/Folk/Choral to Classical/Folk.
+- Server lists have been reorganized to make room for more servers (#875):
+    * Default has been renamed to Any Genre 1.
+	* Any Genre has been renamed to Any Genre 2.
+	* Any Genre 3 has been added.
+	* Choral/Barbershop has been added.
+	* Classical/Folk/Choral has been renamed to Classical/Folk.
 
-- central server addresses are now in jamulus.io instead of fischvolk.de (#919)
+- Central server addresses have been moved to the jamulus.io domain (#919).
+  The fischvolk.de addresses are deprecated.
 
-- show --clientname as first word in title to avoid clipping in Windows task bar (#789)
+- Window title now includes --clientname as the first word to avoid clipping
```

Comment:
If a name is provided using --clientname, that name is shown first in the window title, to avoid clipping in the Windows task bar.

Reply:
Added that as suggested and prepended "UI:" as I can't think of another way to signal the context at the beginning of the line, similar to the other entries.